central/complianceoperator/v2/compliancemanager/utils.go

```diff
@@ -0,0 +1,51 @@
+package compliancemanager
+
+import "github.com/stackrox/rox/generated/internalapi/central"
+
+func buildCreateScanConfig(msgID string, cron string, profiles []string, configName string) *central.MsgToSensor {
```

Comment:
nit: It would be better if you can create a common function to compose a MsgToSensor from a scan request.